lambdas/check-coi/src/main/java/uk/gov/di/ipv/core/checkcoi/CheckCoiHandler.java

```diff
@@ -147,8 +147,8 @@ public Map<String, Object> handleRequest(ProcessRequest request, Context context
                     switch (checkType) {
                         case GIVEN_OR_FAMILY_NAME_AND_DOB -> userIdentityService
                                 .areNamesAndDobCorrelated(combinedCredentials);
-                        case FULL_NAME_AND_DOB -> userIdentityService.areVcsCorrelated(
-                                combinedCredentials);
+                        case FULL_NAME_AND_DOB -> userIdentityService
```

Comment:
Do you think `FULL_NAME_AND_DOB` is still right? I don't think it's necessarily wrong, but it'd be easy to assume that it means the entire name (rather than just a number of chars). On the other hand, it does kinda match the one above in spirit!

Happy to leave, or change, as you prefer.

Reply:
I've gone for `FULL_NAME_WITH_ALLOWANCE_AND_DOB` to match the above and make it more clear it's not an exact match. I'm pretty rubbish at naming things so let me know if you disagree!